tests/e2e/framework/kubernetes.go

```diff
@@ -83,6 +86,7 @@ var (
 	rbacEnable   = flag.Bool("rbac_enable", true, "Enable rbac")
 	localCluster = flag.Bool("use_local_cluster", false,
 		"Whether the cluster is local or not (i.e. the test is running within the cluster). If running on minikube, this should be set to true.")
+	localRegistry             = flag.Bool("use_local_registry", true, "Use in-cluster docker registry for testing")
```

Comment:
We should also update README to clarify how to use `use_local_registry` for e2e test.